tools/src/test/resources/1.7/valid-license-expression-with-text-1.7.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
+<bom serialNumber="urn:uuid:3e671687-395b-41f5-a30f-a58921a69b79" version="1" xmlns="http://cyclonedx.org/schema/bom/1.7">
+    <components>
+        <component type="application">
+            <publisher>Acme Inc</publisher>
+            <group>com.acme</group>
+            <name>tomcat-catalina</name>
+            <version>9.0.14</version>
+            <description>Modified version of Apache Catalina</description>
+            <scope>required</scope>
+            <hashes>
+                <hash alg="MD5">3942447fac867ae5cdb3229b658f4d48</hash>
+                <hash alg="SHA-1">e6b1000b94e835ffd37f4c6dcbdad43f4b48a02a</hash>
+                <hash alg="SHA-256">f498a8ff2dd007e29c2074f5e4b01a9a01775c3ff3aeaf6906ea503bc5791b7b</hash>
+                <hash alg="SHA-512">e8f33e424f3f4ed6db76a482fde1a5298970e442c531729119e37991884bdffab4f9426b7ee11fccd074eeda0634d71697d6f88a460dce0ac8d627a29f7d1282</hash>
+            </hashes>
+            <licenses>
+                <expression bom-ref="my-license" acknowledgement="declared">
```

Comment:
need to test how this can be achieved:
either `<expression>some text</expression>`
or `<expression><value>some text<value></expression>`

well, XML will make it possible :D

Reply:
conslusion:
xml's `mixed=true` with an assert that the number of children must be zero, if `@value` was set.
unfortunately, the assert requires xsd1.1 - which is not implemented widely ...